### DISCUSSION TITLE
Website describes What // How

### DISCUSSION THREAD

**Discussion by startakovsky** [metarank/metarank#321] [Ideas]
Hey, I saw on your website https://www.metarank.ai/ you describe what and how your product works, but you didn't include that excellent blurb in your post that motivates who and why, which I think was excellently summarized. I pasted it below. Just a thought, as I don't know your future plans but it might help in getting folks to reach out to you. Personally this looks like an amazing possible business and AI democratizer. (Promoted.Ai focuses on marketplaces, but this could have its own place in the future, and certainly beats out AWS Personalize.)

_We made a lot of proprietary DIY services for personalization in e-commerce in our past careers and heard so many complaints from other companies also struggling to implement personalization. It’s often considered "too risky" to spend 6+ months on an in-house moonshot project to reinvent the wheel without an experienced team and no existing open-source tools. Like other people in the industry, we were tired of building everything from the bottom up each time we approached personalization - it should be easy not only for Amazon to do such magical ML tricks, but for everyone else._

**Comment by vgoloviznin** [comment]
This makes total sense, we didn't give too much love for the website, but I will take a look at how we can make one of the key selling points more prominent

**Comment by vgoloviznin** [comment]
I've added a similar block to the website!

**Comment by startakovsky** [comment]
Looks great. Good luck in your next steps, wishing you great success!
